src/framework/ui/internal/navigationcontroller.h

```diff
@@ -90,7 +90,7 @@ class NavigationController : public INavigationController, public actions::Actio
     void doActivateSection(INavigationSection* s, bool isActivateLastPanel = false);
     void doDeactivateSection(INavigationSection* s);
     void doActivatePanel(INavigationPanel* s);
-    void doDeactivatePanel(INavigationPanel* s);
+    void doDeactivatePanel(INavigationPanel* panel);
```

Comment:
Better to fix args names in all functions

Reply:
done